Makefile

```diff
@@ -1,58 +1,41 @@
-.PHONY: clean all-images image% launch notebook% package dist run test all-tests test% shell shell% dist-run dist-run% pypi-run pypi-run% mock-server docs-build docs-image version-frontend
+NB_UID := $(shell id -u)
+NB_GID := $(shell id -g)
 
-NB_UID=$(shell id -u)
-NB_GID=$(shell id -g)
-
-IMAGE=rstudio/rsconnect-jupyter-py
+IMAGE := rstudio/rsconnect-jupyter-py
 VERSION := $(shell pipenv run python setup.py --version)
 BDIST_WHEEL := dist/rsconnect_jupyter-$(VERSION)-py2.py3-none-any.whl
 S3_PREFIX := s3://rstudio-connect-downloads/connect/rsconnect-jupyter
-PORT = $(shell printenv PORT || echo 9999)
+PORT := $(shell printenv PORT || echo 9999)
 
 # NOTE: See the `dist` target for why this exists.
 SOURCE_DATE_EPOCH := $(shell date +%s)
 export SOURCE_DATE_EPOCH
 
+.PHONY: clean
 clean:
-	rm -rf build/ dist/ rsconnect_jupyter.egg-info/
+	rm -rf build/ dist/ docs/out/ rsconnect_jupyter.egg-info/
 
-all-images: image2 image3.5 image3.6 image3.7
+.PHONY: all-images
+all-images: image2.7 image3.5 image3.6 image3.7 image3.8
 
 image%:
 	docker build \
 		--tag $(IMAGE)$* \
 		--file Dockerfile \
-		--build-arg BASE_IMAGE=continuumio/miniconda:4.4.10 \
+		--build-arg BASE_IMAGE=python:$*-slim \
 		--build-arg NB_UID=$(NB_UID) \
 		--build-arg NB_GID=$(NB_GID) \
 		--build-arg PY_VERSION=$* \
 		.
 
-launch:
-	docker run --rm -i -t \
-		-v $(CURDIR)/notebooks$(PY_VERSION):/notebooks \
-		-v $(CURDIR):/rsconnect_jupyter \
-		-e NB_UID=$(NB_UID) \
-		-e NB_GID=$(NB_GID) \
-		-e PY_VERSION=$(PY_VERSION) \
-		-p :$(PORT):9999 \
-		$(DOCKER_IMAGE) \
-		/rsconnect_jupyter/run.sh $(TARGET)
-
-
-notebook%:
-	make DOCKER_IMAGE=$(IMAGE)$* PY_VERSION=$* TARGET=run launch
```

Comment:
Is there a recommended way to run for manual testing? `make notebook${VERSION}` used to run the plugin in a notebook in docker with a local mount.

Reply:
@mmarchetti IIUC there is a target that's in the selenium suite that will do this nicely:

```bash
make -C selenium jupyter-up
```